<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoolEyeClass</class>
 <widget class="QMainWindow" name="CoolEyeClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CoolEye</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/liuwei/Desktop/cooleye.png</normaloff>C:/Users/liuwei/Desktop/cooleye.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_image_caml">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Image-CamL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_image_camr">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Image-CamR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_Capture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_Capture_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>600</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>IMU</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbB_uart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_Switch_LR">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>560</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch Left and Right</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1112</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cooleye.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>SL_Caputre()</slot>
 </slots>
</ui>
